config/locales/proper.en.yml

```diff
@@ -285,3 +285,26 @@ en:
         message: "Message:"
         submitted: "Message sent successfully!"
       ctrc_lock: "SCTR Research Nexus services are already present on this request (%{protocol_id}-%{ssr_id}) pertaining to this %{protocol_type}, '%{short_title}'. Please call (843) 792-8688 to edit the existing request."
+      #### short_interaction ####
+      short_interaction:
+        header: "Short Interaction"
+        subject: "Subject of Interaction:*"
+        subject_select: " -- Select Category --"
+        type: "Type of Interaction:* "
+        type_select: "Select Type"
+        duration_placeholder: "Enter an integer"
+        duration_error: "Must enter an integer"
+        duration: "Duration in minutes:*"
+        email_error: "Must enter a valid email address"
```

Comment:
It looks like the various error messages here aren't being used. You can overwrite basic model errors using the `activerecord` hash in `en.yml`. We have a few examples of this already, but here's a link to the docs for this: http://guides.rubyonrails.org/i18n.html#translations-for-active-record-models